File: Java/SivitasAkademik.java
// sivitas akademik adalah manusia

public class SivitasAkademik extends Human
{
	private String asal_universitas;
	private String email_edu;
	
	SivitasAkademik(){
	}
	public void setAsal_universitas(String asal_universitas){
		this.asal_universitas = asal_universitas;
	}
	public String getAsal_universitas(){
		return this.asal_universitas;
	}
	public void setEmail_edu(String email_edu){
		this.email_edu = email_edu;
	}
	public String getEmail_edu(){
		return this.email_edu;
	}
}

File: Python/SivitasAkademik.py
# sivitas akademik adalah manusia

from Human import Human

class SivitasAkademik(Human):
    def __init__(self, __asal_universitas="-", __email_edu="-"):
        super().__init__()
        self.__asal_universitas = __asal_universitas
        self.__email_edu = __email_edu
    
    def setAsal_universitas(self, asal_universitas):
        self.__asal_universitas = asal_universitas

    def getAsal_universitas(self):
        return self.__asal_universitas

    def setEmail_edu(self, email_edu):
        self.__email_edu = email_edu

    def getEmail_edu(self):
        return self.__email_edu
	